docs/database/server_database/sever-database-v1.svg

```diff
@@ -0,0 +1 @@
<text x="13" y="111" class="tf fk">user_id<tspan x="127.38" class="ty">string</tspan></text><path d="M814.545 581.818v-23.273H535.273v-93.09h-46.546v93.09H209.455v139.637H256V605.09h232.727v93.09h46.546v-93.09H768v93.09h46.545V581.819zM674.91 465.455H349.091V139.636h325.818v325.819zM325.82 884.364H139.635V698.182h186.182v186.182z m279.272 0H418.909V698.182h186.182v186.182z m279.273 0H698.182V698.182h186.182v186.182zM395.636 418.909h232.728V186.182H395.636v232.727zM186.182 837.82h93.09v-93.092h-93.09v93.091z m279.273 0h93.09v-93.092h-93.09v93.091z m279.272 0h93.091v-93.092h-93.09v93.091z" class="i ix" transform="translate(83.03625, 100) scale(0.014)"></path></svg><svg width="191.43" height="550" x="-391.67" y="266.99" id="h1537667952" class="t"><rect height="40" width="191.43" class="th"></rect><text x="13" y="25" class="tht">User</text><rect height="510" width="191.43" class="t-bg"></rect><text x="13" y="61" class="tf pk">id<tspan x="178.43" class="ty">string</tspan></text><path d="M7,14A2,2 0 0,1 5,12A2,2 0 0,1 7,10A2,2 0 0,1 9,12A2,2 0 0,1 7,14M12.65,10C11.83,7.67 9.61,6 7, 6A6, 6 0 0, 0 1, 12A6, 6 0 0, 0 7, 18C9.61, 18 11.83, 16.33 12.65, 14H17V18H21V14H23V10H12.65Z" class="i pk" transform="translate(129.08625, 48) scale(0.65)"></path><text x="13" y="86" class="tf">auth_service<tspan x="178.43" class="ty">string</tspan></text><text x="13" y="111" class="tf">delete_at<tspan x="178.43" class="ty">number</tspan></text><text x="13" y="136" class="tf">email<tspan x="178.43" class="ty">string</tspan></text><text x="13" y="161" class="tf">first_name<tspan x="178.43" class="ty">string</tspan></text><text x="13" y="186" class="tf">is_bot<tspan x="178.43" class="ty">boolean</tspan></text><text x="13" y="211" class="tf">is_guest<tspan x="178.43" class="ty">boolean</tspan></text><text x="13" y="236" class="tf">last_name<tspan x="178.43" class="ty">string</tspan></text><text x="13" y="261" class="tf">last_picture_update<tspan x="178.43" class="ty">number</tspan></text><text x="13" y="286" class="tf">locale<tspan x="178.43" class="ty">string</tspan></text><text x="13" y="311" class="tf">nickname<tspan x="178.43" class="ty">string</tspan></text><text x="13" y="336" class="tf">notify_props<tspan x="178.43" class="ty">string</tspan></text><text x="13" y="361" class="tf">position<tspan x="178.43" class="ty">string</tspan></text><text x="13" y="386" class="tf">props<tspan x="178.43" class="ty">string</tspan></text><text x="13" y="411" class="tf">remote_id<tspan x="178.43" class="ty">string?</tspan></text><text x="13" y="436" class="tf">roles<tspan x="178.43" class="ty">string</tspan></text><text x="13" y="461" class="tf">status<tspan x="178.43" class="ty">string</tspan></text><text x="13" y="486" class="tf">timezone<tspan x="178.43" class="ty">string</tspan></text><text x="13" y="511" class="tf">update_at<tspan x="178.43" class="ty">number</tspan></text><text x="13" y="536" class="tf">username<tspan x="178.43" class="ty">string</tspan></text></svg></g></g></svg>
```

Comment:
Not sure if github issue or svg issue, but when trying to see it in github it just show one corner, instead of the full image.

Reply:
I will check this file again

Reply:
The exported SVG does not contain the full diagram.  Hence, we decided to not include it since, we will be using the PNG file.